### ISSUE TITLE
Installed CS_TOOLS, when running the config check  facing error = JSONDecodeError

### ISSUE THREAD

**Issue by CKALA99** [thoughtspot/cs_tools#82]
### First Stop

- [X] I have read and agreed to the ThoughtSpot Dev Tools EULA.
- [X] I understand that if I have a critical issue with ThoughtSpot, I should reach out to my designated Account Executive and Solutions Consultant team instead of cs_tools.
- [X] I searched the cs_tools documentation, with the integrated search.

### Platform Configuration

info snapshot taken on 
![image](https://github.com/thoughtspot/cs_tools/assets/150832577/6b462191-b87f-4339-9ea4-fcd38783ff68)

### Description

I installed CS_Tools and configured the non prod environment file as QA .  I can see the list of config files configured, and see the list of tools available but when I ran the command - cs_tools config check --config QA  , it throwed error " JSONDecodeError : Expecting value : line 1 column 1 (char 0) - This is an unhandled error! " 
![image](https://github.com/thoughtspot/cs_tools/assets/150832577/a3ac66b2-1d5c-47b6-adec-583f170bb6eb)
I also tried this command for invoking rtql command ,similar error.

**Comment by boonhapus** [comment]
Hi @CKALA99 - what version of ThoughtSpot are you on?

**Comment by CKALA99** [comment]
We are in 8.4.1 swcu1 version.

**Comment by boonhapus** [comment]
I'm really not sure what would cause this. Would you be open to meeting on Zoom to check it live? I am in US Central timezone, but can be flexible so we can look together.

Please reach out to me on email: [nicholas.cooper@thoughtspot.com](mailto:nicholas.cooper@thoughtspot.com)

**Comment by CKALA99** [comment]
Thanks a lot Nick , I sent you an invite at 12.45 CST. Please let me know
if you are able to make it .

**Comment by boonhapus** [comment]
After getting on a call with @CKALA99 - we determined that this is network topology related. We attempted to apply proxy settings, attach to VPN, and hit the cluster directly by IP, but to no avail.

This appears to be some other connectivity issue between the ThoughtSpot cluster and the client machine and mostly unrelated to CS Tools. I'll look into adding proxy support formally into the config file, as I think that's a worthwhile improvement, we'll track that one here: #84
